src/field_control_avatar.c

```diff
@@ -42,6 +44,8 @@ static EWRAM_DATA u16 sPrevMetatileBehavior = 0;
 
 u8 gSelectedObjectEvent;
 
+#define NOT_SIGNPOST             0
```

Comment:
```suggestion
#define NOT_SIGNPOST 0
```
would absolutely suffice

Reply:
1. how the eff did you do this from the github UI? I cannot figure it out at all

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/d21d23b52ccd50f1961ebe0ac3c5299692da69cb